### PR TITLE
Editable Content from CMS

### PR DESCRIPTION
**Pull request by Danwhy**
### To be merged after #299 
ref #238: Fetches new content from CMS
 * Uses a Plug to fetch banner text, so it is available on all templates
 * Fetches filter titles, link text for crisis and assessment

### CODE REVIEW

File: test/controllers/banner_test.exs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+defmodule App.BannerTest do
+  use App.ConnCase, async: false
+  doctest App.Banner
```
Comment (shouston3) [inline test/controllers/banner_test.exs]: Very minor point, but could we have `import: true` here (for consistency)

File: web/controllers/banner.ex [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  @moduledoc """
+  # Gets banner text from CMS database, then assigns it to the connection
+  # so it's available in all templates
+  """
+  import Plug.Conn
+  alias App.{Resources}
+
+  @doc"""
+    iex>App.Banner.init(true)
+    true
```
Comment (shouston3) [inline web/controllers/banner.ex]: And then make this:
```elixir
iex> init(true)
true
```

**Review by samhstn — CHANGES_REQUESTED** [review]
Looks great after that small change

**Review by samhstn — APPROVED** [review]
Looks great 👍

**Review by SimonLab — APPROVED** [review]
LGTM :+1:

**Pull request merged**